lib/config-util.js

```diff
@@ -18,18 +18,49 @@ function guessCWD (cwd) {
   return cwd
 }
 
+function moduleExists (name) {
+  try {
+    return typeof require.resolve(name) !== 'undefined'
+  } catch (e) {
+    /* istanbul ignore next */
+    return false
+  }
+}
+
 Config.loadConfig = function (argv, cwd) {
   const rcOptions = [
     argv.nycrcPath || '.nycrc',
     '.nycrc.json',
+    '.nycrc.yml',
+    '.nycrc.yaml',
     'nyc.config.js'
   ]
   const rcPath = findUp.sync(rcOptions, { cwd: guessCWD(cwd) })
   let config = {}
 
   if (rcPath) {
-    if (rcPath.toLowerCase().endsWith('.js')) {
+    const rcPathLower = rcPath.toLowerCase()
+    if (rcPathLower.endsWith('.js')) {
       config = require(rcPath)
+    } else if (/.+\.y(a?)ml$/.test(rcPathLower)) {
+      const installedYamlParser = ['js-yaml', 'yaml'].find((name) => moduleExists(name))
```

Comment:
I'd honestly rather converge on one `yaml` parser, seems like overkill to support multiple flavors.

what's the reason to not just support `js-yaml`, it seems to be the more popular.

Reply:
@isaacs mentioned supporting both in a previous comment